Move getting the storybook package version from a const to a function

FinestV opened the pull request:
- This means the code will only run if used not when the module is imported
- Currently the code runs even if using a storybookUrl configuration.

amitzur commented:
Hey @mcpanzors, thanks for your contribution!
We will shortly release a new major version of this SDK which will allow overriding the batch name by either defining it in the config file or as an env variable, like you implemented.

mcpanzors commented:
This is excellent news. Feel free to close this PR once that branch gets merged into the master

amitzur commented:
I'm glad to share that the new version of Eyes.Storybook SDK is now in beta!
The documentation for beta can be found here:
https://www.npmjs.com/package/@applitools/eyes.storybook/v/2.0.12

Note:
The NPM package is the same as the existing, but with a "beta" flag, so install it with:
npm install @applitools/eyes.storybook@beta

The GitHub repo is currently https://github.com/applitools/eyes.storybook2, and when it's out of beta it will be renamed to the original https://github.com/applitools/eyes.storybook (and this repo will be renamed "eyes.storybook-legacy" or something similar)

To accommodate the requirement in this issue - the default batch name will be the same as the app name, but you may configure it to be something else by either:
1) specify the env var `APPLITOOLS_BATCH_NAME`
2) specify `batchName` property in applitools.config.js

See the [docs](https://www.npmjs.com/package/@applitools/eyes.storybook/v/2.0.12) for more breaking changes

danielputerman commented:
Thanks @mcpanzors. We'll be glad to have your feedback on the new major version.

FinestV committed:
Move getting the storybook package version from a const to a function

This means the code will only run if used not when the module is imported

Currently the code runs even if using a storybookUrl configuration.
